ProcessLib/LiquidFlow/CreateLiquidFlowMaterialProperties.cpp

```diff
@@ -1,105 +0,0 @@
-/**
- * \copyright
- * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
- *            Distributed under a Modified BSD License.
- *              See accompanying file LICENSE.txt or
- *              http://www.opengeosys.org/project/license
- *
- * \file
- *
- *   Created on December 14, 2016, 1:20 PM
- */
-
-#include "CreateLiquidFlowMaterialProperties.h"
-
-#include "BaseLib/Algorithm.h"
-#include "BaseLib/ConfigTree.h"
-#include "MaterialLib/Fluid/FluidProperties/CreateFluidProperties.h"
-#include "MaterialLib/Fluid/FluidProperty.h"
-#include "MaterialLib/Fluid/FluidPropertyHeaders.h"
-#include "MaterialLib/PorousMedium/Permeability/Permeability.h"
-#include "MaterialLib/PorousMedium/Porosity/Porosity.h"
-#include "MaterialLib/PorousMedium/PorousPropertyHeaders.h"
-#include "MaterialLib/PorousMedium/Storage/Storage.h"
-#include "MeshLib/PropertyVector.h"
-
-#include "LiquidFlowMaterialProperties.h"
-
-namespace ParameterLib
-{
-struct ParameterBase;
-}
-
-namespace ProcessLib
-{
-namespace LiquidFlow
-{
-class LiquidFlowMaterialProperties;
-
-std::unique_ptr<LiquidFlowMaterialProperties>
-createLiquidFlowMaterialProperties(
-    BaseLib::ConfigTree const& config,
-    std::vector<std::unique_ptr<ParameterLib::ParameterBase>> const& parameters,
-    MeshLib::PropertyVector<int> const* const material_ids)
-{
-    DBUG("Reading material properties of liquid flow process.");
-
```

Comment:
The corresponding parameter documentations have to been removed as well.

Reply:
Removed.